0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -66,22 +71,25 @@ The SPSP Endpoint MUST respond to HTTPS `GET` requests in the following manner:
 The sender queries the SPSP endpoint to get information about the type of payment that can be made to this receiver:
 
 #### Request
+
 ``` http
 GET /api/spsp/bob HTTP/1.1
 Host: red.ilpdemo.org
-Accept: application/json
+Accept: application/spsp+json
 ```
 
 #### Response
 ``` http
 HTTP/1.1 200 OK
-Content-Type: application/json
+Content-Type: application/spsp+json
 
 {
   "destination_account": "example.ilpdemo.red.bob",
-  "shared_secret": "6jR5iNIVRvqeasJeCty6C-YB5X9FhSOUPCL_5nha5Vs",
-  "maximum_destination_amount": "100000",
-  "minimum_destination_amount": "10",
+  "shared_secret": "6jR5iNIVRvqeasJeCty6C+YB5X9FhSOUPCL/5nha5Vs=",
+  "balance": {
+    "maximum": "100000",
+    "current": "5360"
+  },
```

Comment:
This is a fundamental change, you're  not looking up data about a person, but about an invoice. Although that may be useful by itself, that's not what we use spsp for in ilp-kit.

Can't we keep it so the sender determines the amount, like it's now in ilp-kit? Also, we need to display name and avatar.

Reply:
The `balance` object is only present if this represents an invoice. The sender can still determine the amount; in the case of an invoice it will start to raise the current balance until it reaches the maximum, and if it's just an ordinary receiver then they'll just accept funds no matter what.

The name and avatar URL are still present in the `receiver_info` object. They're not required fields, though, because not everyone may want to display that information.

Reply:
You may want to see the current amount paid before you start trying to pay the invoice, though

Reply:
Is a partially paid invoice something you'd expect to see regularly? It seems like that's more likely to be caused by something going wrong with a chunked payment half-way.

That also reminded me, is there a case to me made for using a specific payment ID in the PSK packet that's related to the invoice, or is the assumption that you would include the type of identifying information in the ILP address that SPSP returns?

Reply:
I think you would probably use the ILP address to encode that information. However, when I upgraded http-ilp for PSK2, I used the payment ID as the `Pay-Token` rather than putting it in the PSK2 application data. Using the ILP address is a third option. We might want to decide which of those is the recommended option.

Reply:
Evan, FWIW, partially paid invoices are a common business scenario, even aside from failed or incomplete payments.